app/views/marc_records/index.json.jbuilder

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+json.array! @marc_records do |marc_record|
```

Comment:
This is 50+ million records in production right now; we probably need to start with a paginated response instead of adding it later.